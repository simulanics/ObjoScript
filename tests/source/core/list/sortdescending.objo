var list = [2,5,1,4,3]
var list2 = ["dog", "house", "airplane", "cat", "goat", "elephant", "bat", "frog"]
list.sortDescending()
list2.sortDescending()

System.print(list)
System.print(list2)