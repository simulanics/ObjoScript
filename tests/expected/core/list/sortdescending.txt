[5, 4, 3, 2, 1]
[house, goat, frog, elephant, dog, cat, bat, airplane]